src/test/kotlin/study/racingcar/domain/CarTest.kt

```diff
@@ -23,6 +23,25 @@ class CarTest {
         assertEquals(0, car.position)
     }
 
+    @Test
+    @DisplayName("자동차 이름이 1자 이상 5자 이하일 때 객체 생성이 성공해야 한다")
+    fun `should create CarName object when name length is between 1 and 5`() {
+        val validNames = listOf("Car1", "Car2", "Car3", "Car4", "Car5")
```

Comment:
테스트 코드를 작성할때 파라미터는 경계값을 기준으로 전달하는것을 권장 드립니다 😄 
```suggestion
        val validNames = listOf("1", "12", "123", "1234", "12345")
```

Reply:
그렇군요! 새로운 것을 배워갑니다 🙂